tests/test_timerange.py

```diff
@@ -18,81 +23,93 @@ def setUp(self):
         mock_dt.fromtimestamp.side_effect = (
             lambda *args, **kw: datetime.datetime.fromtimestamp(*args, **kw)
         )
+        datagrepper.app.app.testing = True
+        self.client = datagrepper.app.app.test_client()
```

Comment:
It does not seem to be used anywhere, it can be removed. If you're looking to create a request context or an app context for the tests, [this is how you can do it](https://flask.palletsprojects.com/en/2.0.x/api/#flask.Flask.test_request_context).